### PR TITLE
Build rdbfmsua.x with a traditional makefile

### LINKED ISSUE #54: Restore rdbfmsua build
The gempak module load had to be disabled due to conflicts with gfs_bufr, disabling the building of `rdbfmsua`. This needs to be resolved to run the full gempak package.

See also #53

### PR DESCRIPTION
**Pull request by aerorahul**
# Description
This PR:
- builds `rdbfmsua.x` with a traditional makefile instead of cmake due to problems with gempak and cmake.  The issues are documented in #50 
- skips building the above executable if the modulefile for this executable is not found.
- only supports wcoss2, hera and orion at the moment where gempak is available.

Fixes #54 

# Type of change
- Maintenance

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
Builds on hera, wcoss2
Runs will be tested at a later time with the workflow.

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: modulefiles/rdbfmsua_wcoss2.intel.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+local gempak_ver=os.getenv("gempak_ver") or "7.14.1"
+
+load(pathJoin("PrgEnv-intel", PrgEnv_intel_ver))
+load(pathJoin("intel", intel_ver))
+load(pathJoin("craype", craype_ver))
+load(pathJoin("cray-mpich", cray_mpich_ver))
+
+load(pathJoin("jasper", jasper_ver))
+load(pathJoin("zlib", zlib_ver))
+load(pathJoin("libpng", libpng_ver))
```
Comment (aerorahul) [inline modulefiles/rdbfmsua_wcoss2.intel.lua]: @WalterKolczynski-NOAA 
Can you test on wcoss with these lines removed?

**Review by WalterKolczynski-NOAA — APPROVED** [review]

File: modulefiles/rdbfmsua_wcoss2.intel.lua (hunk 1 above)
Comment (WalterKolczynski-NOAA) [reply modulefiles/rdbfmsua_wcoss2.intel.lua]: Built without these lines. Update pushed.

File: modulefiles/rdbfmsua_wcoss2.intel.lua [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+local PrgEnv_intel_ver=os.getenv("PrgEnv_intel_ver") or "8.1.0"
+local intel_ver=os.getenv("intel_ver") or "19.1.3.304"
+local craype_ver=os.getenv("craype_ver") or "2.7.10"
+local cray_mpich_ver=os.getenv("cray_mpich_ver") or "8.1.9"
+
+local jasper_ver=os.getenv("jasper_ver") or "2.0.25"
+local zlib_ver=os.getenv("zlib_ver") or "1.2.11"
+local libpng_ver=os.getenv("libpng_ver") or "1.6.37"
+local netcdf_ver=os.getenv("netcdf_ver") or "4.7.4"
```
Comment (aerorahul) [inline modulefiles/rdbfmsua_wcoss2.intel.lua]:
```suggestion
```
These are not needed.

File: modulefiles/rdbfmsua_wcoss2.intel.lua [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
+local bacio_ver=os.getenv("bacio_ver") or "2.4.1"
+local w3emc_ver=os.getenv("w3emc_ver") or "2.9.2"
+local gempak_ver=os.getenv("gempak_ver") or "7.14.1"
+
+load(pathJoin("PrgEnv-intel", PrgEnv_intel_ver))
+load(pathJoin("intel", intel_ver))
+load(pathJoin("craype", craype_ver))
+load(pathJoin("cray-mpich", cray_mpich_ver))
```
Comment (aerorahul) [inline modulefiles/rdbfmsua_wcoss2.intel.lua]: May not need this module either.

Comment (WalterKolczynski-NOAA) [reply modulefiles/rdbfmsua_wcoss2.intel.lua]: Not needed. Updated.

**Pull request merged**